apps/wallet/src/shared/cryptography/keystore.ts

```diff
@@ -2,8 +2,12 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import passworder from '@metamask/browser-passworder';
+
 export async function encrypt(
     password: string,
+    // TODO: is Buffer the best way to pass secrets to passworder?
```

Comment:
The answer to this question is probably not. We can base64 encode it into a string beforehand.

Reply:
We had a discussion with @joyqvq about this - we decided to go with `hex` though